core-bundle/src/Resources/contao/dca/tl_files.php

```diff
@@ -603,16 +603,19 @@ public function checkFilename($varValue, DataContainer $dc)
 		// Check the length without the file extension
 		if ($dc->activeRecord && $varValue)
 		{
-			$intMaxlength = $GLOBALS['TL_DCA'][$dc->table]['fields'][$dc->field]['eval']['maxlength'];
+			$intMaxlength = $GLOBALS['TL_DCA'][$dc->table]['fields'][$dc->field]['eval']['maxlength'] ?? null;
 
-			if ($dc->activeRecord->type == 'file')
+			if ($intMaxlength)
 			{
-				$intMaxlength -= (strlen($dc->activeRecord->extension) + 1);
-			}
+				if ($dc->activeRecord->type == 'file')
+				{
+					$intMaxlength -= (strlen($dc->activeRecord->extension) + 1);
+				}
 
-			if ($intMaxlength && Utf8::strlen($varValue) > $intMaxlength)
-			{
-				throw new Exception(sprintf($GLOBALS['TL_LANG']['ERR']['maxlength'], $GLOBALS['TL_DCA'][$dc->table]['fields'][$dc->field]['label'][0], $intMaxlength));
+				if (Utf8::strlen($varValue) > $intMaxlength)
+				{
+					throw new Exception(sprintf($GLOBALS['TL_LANG']['ERR']['maxlength'], $GLOBALS['TL_DCA'][$dc->table]['fields'][$dc->field]['label'][0], $intMaxlength));
```

Comment:
I know this has been like this before, but shouldn't this be `\Exception`?

(or ideally the more specific `\InvalidArgumentException`)

Reply:
IMO this is not relevant to this PR.

Reply:
Since the file is in the global namespace, the CS fixer would remove the leading backslash anyway.

Reply:
Ah missed the global namespace. You're right oc.